### PR TITLE
Minor clean up code

### PR DESCRIPTION
**Pull request by zh1huang**

### CODE REVIEW

**Review by yuejunfeng0909 — APPROVED** [review]

File: src/test/java/seedu/duke/logic/command/GetCommandTest.java [hunk 1]
```diff
@@ -36,13 +37,20 @@ public void execute_ItemInList_getsNormally() throws Exception {
...
 
     @Test
     public void execute_emptyList_throwsItemNotExistException() {
         assertThrows(ItemNotExistCommandException.class, () -> testCommand2.execute());
     }
 
+    @Test
+    public void execute_nonExistentShelf_throwsShelfNotExistException() {
+        Command testCommand5 = new GetCommand("non","1");
+        assertThrows(ShelfNotExistModelException.class, testCommand5::execute);
```
Comment (t-l-xin) [inline src/test/java/seedu/duke/logic/command/GetCommandTest.java:51]: function name exception thrown can match the actual exception thrown name

File: src/test/java/seedu/duke/logic/command/GetCommandTest.java [hunk 2]
```diff
@@ -36,13 +37,20 @@ public void execute_ItemInList_getsNormally() throws Exception {
         String expected = String.format(GetCommand.GET_COMPLETE_MESSAGE + GetCommand.GET_OUTPUT,
                 "HarryPotter", "16.1", "25.12", Item.DUMMY_ID, "Hello");
         assertEquals(expected, testCommand1.execute());
+        assertEquals(expected, testCommand3.execute());
     }
 
     @Test
     public void execute_emptyList_throwsItemNotExistException() {
         assertThrows(ItemNotExistCommandException.class, () -> testCommand2.execute());
     }
```
Comment (t-l-xin) [inline src/test/java/seedu/duke/logic/command/GetCommandTest.java:46]: exception name in the function can follow throwsItemNotExistCommandException()

**Review by t-l-xin — CHANGES_REQUESTED** [review]
Minor change to test function names

**Review by t-l-xin — APPROVED** [review]
Looks good to merge

**Pull request merged**